### ISSUE TITLE
zbctl-specific migration guide (zbctl to C8 REST API)

### ISSUE THREAD

**Issue by akeller** [camunda/camunda#25584]
## Acceptance criteria

* zbctl-specific migration guide detailing how to migrate to the C8 REST API
  * Supports the following use cases or examples:
    * Debugging deployment issues via desktop/web modeler - zbctl deploy resource my_helpful.bpmn
    * Ruling out network issues between worker and zeebe - zbctl status/create instance/create worker
    * some customers use it as their primary api - deploy, create, activate, complete, etc.
    * To confirm cluster is up & running
    * Get topology, e.g.: for getting replica count, partition distribution, or other related topology info

## Resources

* [Migration guide document](https://docs.google.com/document/d/1h1i2ZzHsrun7lyNVJn6I2Z-VBvAqYwRGcpiN-wmviwM) (🔒 internal)

## Related issues

* Product hub: https://github.com/camunda/product-hub/issues/2524

**Comment by akeller** [comment]
I've added this to both the C8 API and DevEx project boards so we don't lose it.

**Comment by akeller** [comment]
@tmetzke FYI

**Comment by tmetzke** [comment]
Please note that with https://github.com/camunda/camunda/issues/27751 we want to provide a new CLI tool again as the successor of `zbctl`, using the Java client and the REST API.

**Comment by jonathanlukas** [comment]
Related to it, we should remove all references to zbctl from the docs: https://github.com/camunda/camunda-docs/issues/5756

**Comment by akeller** [comment]
FYI @megglos

**Comment by eppdot** [comment]
@tmetzke is this something we still need? 

(cleaning up the finalize api epic after 8.8 has been successfully released)

**Comment by tmetzke** [comment]
Yes, might be something for @megglos to decide how to tackle this, as it's more about the client and developer experience side of things.

**Comment by marcosgvieira** [comment]
🔄 **Status updated automatically (Core Features)**

**From:** No Status  
**To:** Inbox  

**Reason:** Queued for further analysis and triage.

**Comment by lev-safro** [comment]
@megglos is this something you are planning to tackle? I'm moving it to your board, if you have any concerns - please, move it back or reach out to me.

**Comment by megglos** [comment]
I would close this in favor of the plans towards c8ctl. CC @vobu
